awx/main/utils/licensing.py

```diff
@@ -383,12 +384,24 @@ def validate(self):
             current_instances = Host.objects.active_count()
         else:
             current_instances = 0
+        license_date = int(attrs.get('license_date', 0) or 0)
+        if HostMetric:
```

Comment:
The `HostMetric` model from the import should always exist. Won't this always be True?